sierra_adapter/terraform/merger/iam_role_policy.tf

```diff
@@ -3,6 +3,11 @@ resource "aws_iam_role_policy" "vhs_full_access_policy" {
   policy = "${var.vhs_full_access_policy}"
 }
 
+resource "aws_iam_role_policy" "allow_read_from_windows_q" {
```

Comment:
Either this name is wrong, or the queue is wrong.

Reply:
name is wrong